sdk/communication/azure-communication-identity/tests/test_communication_identity_client.py

```diff
@@ -250,16 +250,20 @@ def test_delete_user(self, communication_livetest_dynamic_connection_string):
         assert user.properties.get('id') is not None
 
     @CommunicationPreparer()
+    @recorded_by_proxy
     def test_create_user_and_token_with_no_scopes(self, communication_livetest_dynamic_connection_string):
         identity_client = CommunicationIdentityClient.from_connection_string(
             communication_livetest_dynamic_connection_string,
             http_logging_policy=get_http_logging_policy()
         )
 
         with pytest.raises(Exception) as ex:
-            user, token_response = identity_client.create_user_and_token(scopes=None)
+            identity_client.create_user_and_token(scopes=None)
+
+        assert ex is not None
```

Comment:
I'm not sure if this assertion adds much since we know we caught an exception by this point, but it also doesn't hurt. Is there any consistent content in the exception message -- or a more specific type than `Exception` -- that we can assert for instead?

Reply:
Thanks! That's a good point. I updated it.

Reply:
do we really need to assert the exeption for nonnullability and the message value? what if the auth team will change the exception message, so it means we have to update our tests each time the auth team update the message from service side? :)

Reply:
As there are several possible exceptions we are checking for and we need a way to distinguish them: I had an issue of getting the "wrong" exception in these tests and spending a lot of time to find a problem when the test was green. Checking the error message is a trade-off here as there seems to be no constant property to distinguish between exceptions raised for different reasons. 
Updating the tests if Auth team changes the wording shouldn't take much time.
Please share if you have an idea how else we could check the kind of exception we're getting here.

Reply:
I would prefer to check only for the response status code and non-emptiness of the error message, as we did for the [async tests](https://github.com/Azure/azure-sdk-for-python/blob/e778c87f176bd1a44b6cc6adbbf68ab0caa108c5/sdk/communication/azure-communication-identity/tests/test_communication_identity_client_async.py#L141), but if others are fine with your approach then we can ignore my comment here

Reply:
Let's stick to the approach from async tests than. I've updated